src/Joomlatools/Console/Command/Vhost/Create.php

```diff
@@ -144,4 +127,66 @@ protected function execute(InputInterface $input, OutputInterface $output)
             @unlink($tmp);
         }
     }
+
+    protected function _getVariables(InputInterface $input)
+    {
+        $documentroot = Util::isPlatform($this->target_dir) ? $this->target_dir . '/web/' : $this->target_dir;
+
+        $variables = array(
```

Comment:
@stevenrombauts genius move here, naming the variables here in such a manner makes the template files so much more readable.